reference.md

```diff
@@ -0,0 +1,1393 @@
+# Snapcraft CLI reference
+
+## lifecycle and misc
+
+### version
+
+    $ snapcraft --version
+    2.12.1
+
+### help
+
+    $ snapcraft help
+    Usage:
+      snapcraft [options] [--enable-geoip --no-parallel-build]
+      snapcraft [options] init
+      snapcraft [options] pull [<part> ...]  [--enable-geoip]
+      snapcraft [options] build [<part> ...] [--no-parallel-build]
+      snapcraft [options] stage [<part> ...]
+      snapcraft [options] prime [<part> ...]
+      snapcraft [options] strip [<part> ...]
+      snapcraft [options] clean [<part> ...] [--step <step>]
+      snapcraft [options] snap [<directory> --output <snap-file>]
```

Comment:
I think this should be [<directory>] [--output <snap-file>]. I'm not sure if that will make docopt go crazy, need to try it.